tests/ui/test-attrs/test-type.rs

```diff
@@ -1,4 +1,4 @@
-// compile-flags: --test
+// compile-flags: --test -Zpanic-abort-tests
```

Comment:
this test is `needs-unwind` already, is this needed? would it make sense to create a second test that isn't `needs-unwind` and add `-Zpanic-abort-tests` there?

Reply:
I originally made and tested this commit on a branch that didn't have https://github.com/rust-lang/rust/pull/111174, and I had to pass that flag to make the test work without unwinding. @bjorn3 is there a reason you added `needs-unwind` rather than adding the flag?

IMO adding the flag and removing the `needs-unwind` is the right fix here.

Reply:
I guess I didn't think of adding `-Zpanic-abort-tests`.

Reply:
Ok removed `needs-unwind` from the test, good catch @Nilstrieb!